axon/controller/axon_rpyc_controller.py

```diff
@@ -84,7 +90,8 @@ def __init__(self):
         self.exposed_stats = exposed_Stats()
         self.exposed_namespace = exposed_Namespace()
         self.exposed_interface = exposed_Interface()
-        self.resource_monitor = ResourceMonitor(self._record_queue)
+        self.exposed_resource_monitor = exposed_ResourceMonitor(self._record_queue)
+        self.exposed_console = exposed_Console()
```

Comment:
Let's avoid this as well. 
Let Console application be only a base class for other apps but not allow to run any commands from user for now unless we add input validation.